crates/iota-graphql-rpc/Cargo.toml

```diff
@@ -95,10 +95,8 @@ iota-test-transaction-builder.workspace = true
 simulacrum.workspace = true
 
 [features]
-default = ["pg_backend", "postgres-feature"]
-postgres-feature = ["diesel/postgres", "diesel/postgres_backend"]
+default = []
```

Comment:
This can be removed since we have no defaults now

Reply:
Done in ede80356014e2f7cf8a1b8ed3034837f9104d872